main.rb

```diff
@@ -0,0 +1,62 @@
+require_relative './app'
+
+def make_person
```

Comment:
- [ ] When running your project I am getting the following error 👇  please fix it or add more info on how to run your project please

![image](https://user-images.githubusercontent.com/75126481/225756490-e740087f-19c8-4917-86ae-210e6df5ace4.png)

![image](https://user-images.githubusercontent.com/75126481/225756827-37da2197-c0a7-450b-89c1-4b48be6c1bc0.png)

Reply:
I have added run terminal instructions. Thanks